source/extensions/filters/http/grpc_json_transcoder/transcoder_input_stream_impl.cc

```diff
@@ -9,6 +9,8 @@ int64_t TranscoderInputStreamImpl::BytesAvailable() const { return buffer_->leng
 
 bool TranscoderInputStreamImpl::Finished() const { return finished_; }
 
+int64_t TranscoderInputStreamImpl::bytesStored() const { return buffer_->length(); }
```

Comment:
unint64_t  or std::size_t?   is it ever be -1?

Reply:
Done. `uint64_t` is correct.